### PR TITLE
add rotation degree property

### PR DESCRIPTION
**Pull request by xxminamixx**
Added as a property so that the degree of rotation can be customized.

### CODE REVIEW

**Comment by AndrewSB** [comment]
Awesome, thanks a bunch! Could you add an entry in the README under customization for this new property? In this section: https://github.com/zhxnlai/ZLSwipeableViewSwift#customization

**Comment by xxminamixx** [comment]
@AndrewSB 
Thank you for the review.
I will contact you again if I fix it!

**Comment by xxminamixx** [comment]
@AndrewSB 
I added angle property to README
Please check!

File: README.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ Here is a list of customizable behaviors:
 public var animateView = ZLSwipeableView.defaultAnimateViewHandler()
 public var interpretDirection = ZLSwipeableView.defaultInterpretDirectionHandler()
 public var shouldSwipeView = ZLSwipeableView.defaultShouldSwipeViewHandler()
+public var angle = CGFloat(1.0)
```
Comment (AndrewSB) [inline README.md:139]: cool! could you also add an entry at line 160 of the README with an explanation for how it works?

P.S.: this commit doesn’t link to your github profile, if that’s something you care about I’d recommend updating the git config on the machine you’re working on to include your github email address!

**Comment by xxminamixx** [comment]
@AndrewSB 

I added angle property description.

> P.S.: this commit doesn’t link to your github profile, if that’s something you care about I’d recommend updating the git config on the machine you’re working on to include your github email address!

Thank you for teaching me!

**Comment by AndrewSB** [comment]
Of course! 

This LGTM now, but your first two commits aren't attributed to you -- you can `git commit --amend` them to change your email in the author field, or I can merge as-is. Just let me know and I'll proceed. 

Here's a guide if you're interested: https://www.git-tower.com/learn/git/faq/change-author-name-email, I'd recommend using the interactive rebase they mention if you'd like to update the first two commits in this PR

**Comment by xxminamixx** [comment]
@AndrewSB 

Thank you for the review!
Fixed a previous commit.

If you have anything else please contact me

**Comment by AndrewSB** [comment]
awesome!

**Pull request merged**